docs/evaluation/code.md

```diff
@@ -0,0 +1,30 @@
+# Code
+
+More details are coming soon!
+
+## Supported benchmarks
+
+### swe-bench
+
+- Benchmark defined in [`nemo_skills/dataset/swe-bench/__init__.py`](https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/dataset/swe-bench/__init__.py)
+- Original benchmark source is [here](https://github.com/SWE-bench/SWE-bench).
+
+### livecodebench
```

Comment:
Adding this comment as a reminder for the "More details" part.

For livecodebench, specifying the split is quite important since there are so many slices of the data. This can affect the preparation as well i.e. not covered by default preparation setup.